components/playlist/browser/resources/reducers/playlist_reducer.ts

```diff
@@ -17,7 +17,12 @@ const playlistReducer: Reducer<PlaylistData | undefined> = (
   action
 ) => {
   if (state === undefined) {
-    state = { lists: [], currentList: undefined, lastPlayerState: undefined }
+    state = {
+      lists: [],
+      currentList: undefined,
+      lastPlayerState: undefined,
+      cachingProgress: undefined
```

Comment:
Might be worth setting this to `new Map` (and updating the type definition). Then we'd never need to worry about whether it's defined or not :smile: